src/components/NcRichText/helpers.js

```diff
@@ -3,14 +3,12 @@
  *
  * @type {RegExp}
  */
-export const URL_PATTERN = /(\s|^)(https?:\/\/)((?:[-A-Z0-9+_]+\.)+[-A-Z]+(?:\/[-A-Z0-9+&@#%?=~_|!:,.;()]*)*)(\s|$)/ig
+export const URL_PATTERN = /(\s|^)(https?:\/\/)([-A-Z0-9+_.]+(?::[0-9]+)?(?:\/[-A-Z0-9+&@#%?=~_|!:,.;()]*)*)(\s|$)/ig
+export const RN_AUTOLINK = /(\s|^)(https?:\/\/)([-A-Z0-9+_.]+(?::[0-9]+)?(?:\/[-A-Z0-9+&@#%?=~_|!:,.;()]*)*)(?=\s|\)|$)/ig
```

Comment:
I don't understand what this is for. Seems very similar to the `URL_PATTERN` above. What does RN stand for?

Reply:
oh... and this is also not being tested. You might want to add it to the test to prevent regressions here as well.

Reply:
Oh, good catch. This was a leftover - removed now.